tests/lib/helpers.js

```diff
@@ -218,6 +232,13 @@ function assertUATrackingPageVar(test, key, val) {
 }
 
 
+function assertNotVisible(selector) {
```

Comment:
Fwiw this is the code casper uses: https://github.com/n1k0/casperjs/blob/master/modules/clientutils.js#L156